concrete/src/Area/Layout/Layout.php

```diff
@@ -183,11 +183,6 @@ abstract public function exportDetails($node);
     public function export($node)
     {
         $layout = $node->addChild('arealayout');
-        if ($this->isAreaLayoutUsingThemeGridFramework()) {
```

Comment:
Yes – I'm not absolutely 100% positive but I'm 99% certain that that type check could just be moved into the subclasses instead of being in Layout.php.

Reply:
No, I think you're right - I'd definitely prefer to have it all be in the sublcasses. I'm just wondering if there was some weird reason why this code isn't in the subclasses, like sometimes layouts get screwed up and don't remember if they're ThemeGrid or CustomLayout. But I don't think that's the case. I think we can move it into the subclasses and get rid of the "if"

Reply:
Great! So, this PR should be ready :wink: